versions.mk

```diff
@@ -28,6 +28,7 @@ VERSION  ?= v25.3.0-rc.2
 vVERSION := v$(VERSION:v%=%)
 
 GOLANG_VERSION := $(shell ./hack/golang-version.sh)
+TOOLKIT_CONTAINER_IMAGE := $(shell ./hack/toolkit-container-image.sh)
```

Comment:
in the future we may want to rename `hack` to something  slightly more worthy :)